x-pack/plugins/dataset_quality/server/routes/data_streams/routes.ts

```diff
@@ -135,8 +136,36 @@ const dataStreamDetailsRoute = createDatasetQualityServerRoute({
   },
 });
 
+const estimatedDataInBytesRoute = createDatasetQualityServerRoute({
+  endpoint: 'GET /internal/dataset_quality/data_streams/estimated_data',
```

Comment:
Nice one adding `x-pack/plugins/dataset_quality/server/routes/data_streams/get_estimated_data_in_bytes/get_estimated_data_in_bytes.test.ts`, do we also need an API test?